.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,13 @@
+contact_links:
+  - name: Ask on Telegram (Russian)
+    url: https://t.me/gotd_ru
+    about: You can ask for help in Russian here!
+  - name: Ask on Telegram (English)
+    url: https://t.me/gotd_en
+    about: You can ask for help in English here!
+  - name: Ask on GitHub
+    url: https://github.com/tie/td/discussions/new?category=q-a
```

Comment:
```suggestion
    url: https://github.com/gotd/td/discussions/new?category=q-a
```

Reply:
Whoops, I’ve totally missed that! Force-pushed a change so we won’t have my URL in commit history.

```diff
diff --git a/.github/ISSUE_TEMPLATE/config.yml b/.github/ISSUE_TEMPLATE/config.yml
index 09a5647b..ed7cde7d 100644
--- a/.github/ISSUE_TEMPLATE/config.yml
+++ b/.github/ISSUE_TEMPLATE/config.yml
@@ -6,8 +6,8 @@ contact_links:
     url: https://t.me/gotd_en
     about: You can ask for help in English here!
   - name: Ask on GitHub
-    url: https://github.com/tie/td/discussions/new?category=q-a
-    about: You can ask for help without using Telegram here!
+    url: https://github.com/gotd/td/discussions/new?category=q-a
+    about: You can ask for help without using Telegram too!
   - name: Want to contribute to gotd?
     url: https://github.com/gotd/td/blob/main/CONTRIBUTING.md
     about: Be sure to read contributing guidelines!
```